flink-core/src/test/java/org/apache/flink/util/AbstractIDTest.java

```diff
@@ -71,16 +67,16 @@ public void testCompare() throws Exception {
         AbstractID id10 = new AbstractID(Long.MIN_VALUE, Long.MAX_VALUE);
 
         // test self equality
-        assertEquals(0, id1.compareTo(CommonTestUtils.createCopySerializable(id1)));
-        assertEquals(0, id2.compareTo(CommonTestUtils.createCopySerializable(id2)));
-        assertEquals(0, id3.compareTo(CommonTestUtils.createCopySerializable(id3)));
-        assertEquals(0, id4.compareTo(CommonTestUtils.createCopySerializable(id4)));
-        assertEquals(0, id5.compareTo(CommonTestUtils.createCopySerializable(id5)));
-        assertEquals(0, id6.compareTo(CommonTestUtils.createCopySerializable(id6)));
-        assertEquals(0, id7.compareTo(CommonTestUtils.createCopySerializable(id7)));
-        assertEquals(0, id8.compareTo(CommonTestUtils.createCopySerializable(id8)));
-        assertEquals(0, id9.compareTo(CommonTestUtils.createCopySerializable(id9)));
-        assertEquals(0, id10.compareTo(CommonTestUtils.createCopySerializable(id10)));
+        assertThat(id1.compareTo(CommonTestUtils.createCopySerializable(id1))).isZero();
```

Comment:
```suggestion
        assertThat(id1).isEqualByComparingTo(CommonTestUtils.createCopySerializable(id1));
```

The rest can be updated as well